app/models/product.rb

```diff
@@ -0,0 +1,9 @@
+class Product < ApplicationRecord
+  belongs_to :user
+
+  validates :name, :eth_address, :create_bot_product_tx, presence: true
+
+  def as_json(options={})
```

Comment:
where do we use this?

Reply:
we used this [here](https://github.com/TallaBotChain/botchain-example-service/pull/42/files#diff-4676c008b11a5480d73d4a6de01e45b9R14) and then [here](https://github.com/TallaBotChain/botchain-example-service/pull/42/files#diff-9599427925097c3c66f26ac1e0de5cadR28) And then used this as initial state for developer and products